app/retail/templates/emails/grants/subscription_terminated.txt

```diff
@@ -1,21 +1,21 @@
 {% load i18n humanize %}
 
-{% trans "Grant" %} {{grant_name}} {% trans "has been cancelled" %}
+{% trans "Grant" %} {{ grant_name }} {% trans "has been cancelled" %}
```

Comment:
All of these variables need updated to take the subscription object and use it's associated attributes.